app/mailers/invoice_mailer.rb

```diff
@@ -10,6 +10,7 @@ def finalized
 
     return if @organization.email.blank?
     return if @customer.email.blank?
+    return if @invoice.fees_amount_cents.zero?
```

Comment:
Can we add a spec for this case?

Reply:
Yes sorry it's draft I forgot to put it in draft

Reply:
done @ivannovosad !